.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,6 +5,6 @@ Closes: ??
 ### Checklist
 
 - [ ] Changes covered by tests (tests added if needed)
-- [ ] PR submitted to [blitzjs.com](https://github.com/blitz-js/blitzjs.com) for any user facing changes
+- [ ] PR requests documentation changes on [blitzjs.com](https://github.com/blitz-js/blitzjs.com)
```

Comment:
Thanks! That specific wording is a bit confusing I think. What do you think about this?

```suggestion
- [ ] Does this PR warrant documentation updates? If yes, open a second PR with doc changes to [blitzjs.com](https://github.com/blitz-js/blitzjs.com)
```

Reply:
Yeah seems nice